src/store/reducers/tenants/utils.ts

```diff
@@ -174,9 +165,8 @@ const calculateTenantEntities = (tenant: TTenant) => {
     return {nodesCount, groupsCount};
 };
 
-export const prepareTenants = (tenants: TTenant[], useNodeAsBackend: boolean): PreparedTenant[] => {
+export const prepareTenants = (tenants: TTenant[]): PreparedTenant[] => {
```

Comment:
As I see we used to use nodeId for backend - this was determined by singleClusterMode in tenants.ts

Why we are not using this approach now?

Reply:
Oh I see - we are not using tenant.backend field somewhere above...